build.gradle.kts

```diff
@@ -329,9 +331,7 @@ allprojects {
             apply<CheckstylePlugin>()
             checkstyle {
                 toolVersion = "checkstyle".v
-                configProperties = mapOf(
-                    "cache_file" to buildDir.resolve("checkstyle/cacheFile")
-                )
+                configDirectory.set(rootProject.file("config/checkstyle"))
```

Comment:
I think this change was good for learning but let's not upstream this change right now.

Reply:
I removed this change